ng-dev/release/notes/context.ts

```diff
@@ -49,6 +50,14 @@ export class RenderContext {
 
   constructor(private readonly data: RenderContextData) {}
 
+  /** Gets the group name for the given commit. */
+  private _commitToGroup(commit: CommitFromGitLog): string {
+    if (this.data.commitToGroup) {
+      return this.data.commitToGroup(commit);
+    }
+    return commit.scope;
+  }
```

Comment:
Previously the only usage of `data` was during construction, setting up the values of the class members.  I don't know if it's worth keeping that pattern for this or not.

```suggestion
  private _commitToGroup = this.data.commitToGroup || (commit: CommitFromGitLog) => commit.scope;
```

Reply:
yeah, I was thinking about something like this (that's why there is the `defaultCommitToGroupFn` leftover. I think it's more clear in a function though. We use `this.data` in some functions below as well.